st_files_connection/connection.py

```diff
@@ -67,11 +77,15 @@ def _connect(self, **kwargs) -> "AbstractFileSystem":
     
     @property
     def fs(self) -> "AbstractFileSystem":
+        """Access the underlying AbstractFileSystem for full API operations."""
         return self._instance
 
     def open(
         self, path: str | Path, mode: str = "rb", *args, **kwargs
     ) -> Iterator[TextIOWrapper | AbstractBufferedFile]:
+        """
+        Open the specified path as a file-like object
+        """
```

Comment:
nit: conventionally, the docstring summary should start on the same line as the opening triple-quotes.